tests/conftest.py

```diff
@@ -421,3 +431,45 @@ def archive():
 
     except Exception as e:
         warnings.warn(str(e))
+
+
+# Get the flat, fringe, and linearity for
+# a couple of DECam chip sand filters
+# Need session scope; otherwise, things
+# get mixed up when _get_default_calibrator
+# is called from within another function.
+@pytest.fixture( scope='session' )
+def decam_default_calibrators():
+    decam = get_instrument_instance( 'DECam' )
+    sections = [ 'N1', 'S1' ]
+    filters = [ 'r', 'i', 'z' ]
+    for sec in sections:
+        for calibtype in [ 'flat', 'fringe' ]:
+            for filt in filters:
+                decam._get_default_calibrator( 60000, sec, calibtype=calibtype, filter=filt )
+    decam._get_default_calibrator( 60000, sec, calibtype='linearity' )
```

Comment:
I feel like we should have celebrated more this monumentous date when we got to live through it.

Reply:
My housemate in grad school had a "10 kiloday" party when he was 10,000 days old.